keyboards/helix/rev2/keymaps/default/keymap.c

```diff
@@ -11,7 +11,6 @@
   #include "ssd1306.h"
 #endif
 
```

Comment:
Theres still a change to the helix default keymap. This GitHub suggestion might work? but not 100% sure as it wont directly let you suggest for deleted lines.

```suggestion

extern keymap_config_t keymap_config;
```